Modelica/Mechanics/Translational.mo

```diff
@@ -4943,6 +5001,130 @@ If the heatPort is not used (useHeatPort = false), the operational temperature r
 However, the speed v_nominal at which the maximum torque occurs is adapted from reference temperature TRef to the operational temperature.</p>
 </html>"));
     end EddyCurrentForce;
+
+    model DrivingResistance "Simple model of driving resistances"
+      parameter Real cw(start=0.5) "Drag resistance coefficient"
+        annotation(Dialog(group="Drag resistance"));
+      parameter Modelica.SIunits.Density rho=1.18 "Density of air"
+        annotation(Dialog(group="Drag resistance"));
+      parameter Modelica.SIunits.Area A(start=1) "Cross section of vehicle"
+        annotation(Dialog(group="Drag resistance"));
+      parameter Boolean useWindInput=false "Use wind input?"
+        annotation(Dialog(group="Drag resistance"));
+      parameter Modelica.SIunits.Velocity vWindConstant=0 "Constant wind velocity"
+        annotation(Dialog(group="Drag resistance", enable=not useWindInput));
+      parameter Boolean usecrInput=false "Use cr input?"
+        annotation(Dialog(group="Rolling resistance"));
+      parameter Real crConstant(start=0.1) "Constant rolling resistance coefficient"
+        annotation(Dialog(group="Rolling resistance", enable=not usecRInput));
```

Comment:
This should be `enable= not usecrInput`.

Reply:
done

Reply:
> 
> 
> This should be `enable= not usecrInput`.

What I meant ist that in the enable condition the variable should be named usecRInput (instead of usecrInput). I believe you understood it as renaming all Boolean parameters from usexyz to enablexyz, which was not my intention.

Reply:
For example, the Boolean parameter is named `enablecrInput`, but in the Dialog.enable annotation `usecRInput` is referenced. This should be consistent.

Generally, I am fine with the useXYZ naming.